src/ncp/ncp_base_mtd.cpp

```diff
@@ -4571,6 +4591,27 @@ void NcpBase::HandleUdpForwardStream(otMessage *aMessage, uint16_t aPeerPort, ot
 }
 #endif // OPENTHREAD_CONFIG_UDP_FORWARD_ENABLE
 
+int NcpBase::HandleCliOutput_Jump(void *aContext, const char *aFormat, va_list aArguments)
+{
+    return static_cast<NcpBase *>(aContext)->HandleCliOutput(aFormat, aArguments);
+}
+
+int NcpBase::HandleCliOutput(const char *aFormat, va_list aArguments)
+{
+    uint8_t header = SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0;
+    char    output[OPENTHREAD_CONFIG_CLI_LOG_INPUT_OUTPUT_LOG_STRING_SIZE];
```

Comment:
I think the size here should be `OPENTHREAD_CONFIG_CLI_MAX_LINE_LENGTH ` rather than ` OPENTHREAD_CONFIG_CLI_LOG_INPUT_OUTPUT_LOG_STRING_SIZE`. The `OPENTHREAD_CONFIG_CLI_LOG_INPUT_OUTPUT_LOG_STRING_SIZE` is the log string buffer size.